patches/chromium/printing.patch

```diff
@@ -204,19 +214,27 @@ index 7ba43aada1ac44827cca264d6f37814e4a91f458..c41b0c24974147e847baa21b9b489261
  }
  
 diff --git a/chrome/browser/printing/print_view_manager_base.h b/chrome/browser/printing/print_view_manager_base.h
-index cf074791d0e2e17bbf8cf0b000b8d63e235b7deb..c12107d0af1291c113e05bc1a9cc87e2466c8610 100644
+index cf074791d0e2e17bbf8cf0b000b8d63e235b7deb..30b6e042b12f23299b3b7eaecb8a8e71c221786d 100644
 --- a/chrome/browser/printing/print_view_manager_base.h
 +++ b/chrome/browser/printing/print_view_manager_base.h
-@@ -39,6 +39,8 @@ class PrintJob;
+@@ -21,6 +21,7 @@
+ #include "content/public/browser/notification_observer.h"
+ #include "content/public/browser/notification_registrar.h"
+ #include "printing/buildflags/buildflags.h"
++#include "v8/include/v8.h"
+ 
+ struct PrintHostMsg_DidPrintDocument_Params;
+ 
+@@ -39,6 +40,8 @@ class PrintJob;
  class PrintQueriesQueue;
  class PrinterQuery;
  
-+using CompletionCallback = base::OnceCallback<void(bool)>;
++using CompletionCallback = base::OnceCallback<void(bool, v8::Local<v8::Value>)>;
```

Comment:
Should just pass `const std::string&`, V8 conversation would be done automatically.